### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag @jframos 
#### Description

Solves issue 4874
#### Testing

N/A

### CODE REVIEW

File: fiware-region-sanity-tests/README.rst [hunk 1]
```diff
@@ -58,94 +58,103 @@ Test Algorithm
...
-                "PiraeusU": "public-ext-net-01",
-                "Zurich": "public-ext-net-01",
-                "Karlskrona": "public-ext-net-01",
-                "Volos": "public-ext-net-01",
-                "Budapest": "public-ext-net-01",
-                "Stockholm": "public-ext-net-01",
-                "SophiaAntipolis": "net04_ext",
-                "Poznan": "public-ext-net-01",
-                "Gent": "public-ext-net-03",
-                "Crete": "public-ext-net-01"
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst:148]: If you delete this value make a reference to the document in which a user can find the values due to they are using it like reference to understand what they want to do.

File: fiware-region-sanity-tests/README.rst [hunk 2]
```diff
@@ -155,42 +164,27 @@ All configuration values will be 'strings'.
...
+* Go to the root folder of the project.
+* Run ``launch_tests.sh``. This command will execute all sanity tests in all
+  regions found under ``tests/regions/`` folder:
 
-  nosetests tests.regions.test_spain --exe \
-          --with-xunit --xunit-file=test_results.xml \
-          --with-html --html-report=test_results.html \
-          --html-report-template=resources/templates/test_report_template.html -v
+  - It is possible to provide a list of regions (in lowercase) as argument to
+    restrict the execution to them
```
Comment (flopezag) [inline fiware-region-sanity-tests/README.rst:172]: example?

File: fiware-region-sanity-tests/README.rst (hunk 1 above)
Comment (pratid) [inline fiware-region-sanity-tests/README.rst:148]: Deleted on purpose not to duplicate actual data found in "resources/settings.json", which is aforementioned.

Comment (flopezag) [inline fiware-region-sanity-tests/README.rst:148]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**